synapse/config/logger.py

```diff
@@ -138,6 +137,12 @@
 removed in Synapse 1.3.0. You should instead set up a separate log configuration file.
 """
 
+STRUCTURED_ERROR = """\
+Support for the structured configuration option was removed in Synapse 1.54.0.
+You should instead use the standard logging configuration. See
+https://matrix-org.github.io/synapse/v1.52/structured_logging.html
```

Comment:
```suggestion
https://matrix-org.github.io/synapse/v1.54/structured_logging.html
```

So the link doesn't point to a doc saying that legacy configs will be automatically converted.

Reply:
Thanks! I originally removed the info completely so wanted to link to an old version which had it still, but then realized we should leave the conversion notes for a little while longer.